tomlkit/container.py

```diff
@@ -46,42 +46,33 @@ def __init__(self, parsed: bool = False) -> None:
     def body(self) -> List[Tuple[Optional[Key], Item]]:
         return self._body
 
-    def unwrap(self, recursive: bool = True) -> str:
-        unwrapped = {}
-        for k, v in self._body:
-            if k is None:
-                continue
+    def _unwrap_inner(self, k, v, d):
+        if k is None:
+            return
 
-            k = k.key
+        k = k.key
+        v = v.value
+
+        if isinstance(v, Container):
```

Comment:
It is safe to use `isinstance(v, Item)` here since the child of a container can't be a container.

Reply:
I tried that, not sure I did it correctly